src/java.base/share/classes/java/lang/AbstractStringBuilder.java

```diff
@@ -1821,4 +1821,98 @@ private final void appendChars(CharSequence s, int off, int end) {
         }
         count += end - off;
     }
+
+    /**
+     * Appends {@code count} copies of the character {@code c} to this sequence.
+     * <p>
+     * The length of this sequence increases by {@code count}.
+     *
+     * @param c      character to append
+     * @param count  number of times to copy
+     *
+     * @return  a reference to this object.
+     *
+     * @since 21
```

Comment:
\@since should be last / after \throws; here and in the other method added.

Reply:
Changed